tests/integration/test_client_warnings_banner/test.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+import helpers.cluster.ClickHouseCluster
+import helpers.client.Client
+
+cluster = helpers.cluster.ClickHouseCluster(__file__)
+
+node = cluster.add_instance('node')
+
+@pytest.fixture(scope="module", autouse=True)
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+def test_default(started_cluster):
+    try:
+        output = node.exec_in_container(
+            ['sh', '-c', '/usr/bin/clickhouse client -q "SELECT 1"'],
+            privileged = True,
+            user = 'root',
+        )
+    except AssertionError:
+        raise
+    except Exception as ex:
+        print(ex)
+    assert 'Warnings:' in output
+
+def test_cli_no_warnings(started_cluster):
```

Comment:
We tend to avoid integration tests as much as possible. We can mostly accomplish the same using a stateless test i.e. a `.sql` or `.sh` test (there are examples in the repo).